File: Selenium/Activities/Activity19.java
package activities;


import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity19 {
    public static void main(String[] args) {
    	WebDriver driver = new ChromeDriver();
    	
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");
        System.out.println("Home page title: " + driver.getTitle());

        driver.findElement(By.id("confirm")).click();

        Alert confirmAlert = driver.switchTo().alert();
        String alertBoxMessage = confirmAlert.getText();
        System.out.println("Text in alert box is: " + alertBoxMessage);

        confirmAlert.accept();

        driver.quit();
    }
}

File: Selenium/Activities/activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/javascript-alerts")
    print("Page title is: ", driver.title)

    driver.find_element(By.ID, "confirm").click()

    confirm_alert = driver.switch_to.alert
    print(confirm_alert.text)

    confirm_alert.accept()